cmd/postgres-operator/main.go

```diff
@@ -120,20 +120,25 @@ func addControllersToManager(ctx context.Context, mgr manager.Manager) error {
 func isOpenshift(ctx context.Context, cfg *rest.Config) bool {
 	log := logging.FromContext(ctx)
 
-	const sccGroup, sccKind = "security.openshift.io", "SecurityContextConstraints"
+	const sccGroupName, sccKind = "security.openshift.io", "SecurityContextConstraints"
 
 	client, err := discovery.NewDiscoveryClientForConfig(cfg)
 	assertNoError(err)
 
-	_, resourceLists, err := client.ServerGroupsAndResources()
-	assertNoError(err)
-
-	// If we detect that the "SecurityContextConstraints" Kind is present in the
-	// "security.openshift.io" Group, we'll return that this is an OpenShift
-	// environment
-	for _, rl := range resourceLists {
-		if strings.HasPrefix(rl.GroupVersion, sccGroup+"/") {
-			for _, r := range rl.APIResources {
+	groups, err := client.ServerGroups()
```

Comment:
⛏️ The variable for the other calls is `resourceList`.

```suggestion
	groupList, err := client.ServerGroups()
```